.github/workflows/05-prepare-release.yml

```diff
@@ -56,12 +56,13 @@ jobs:
 
   check-recipe:
     runs-on: ubuntu-24.04
-    if: github.repository == 'shopware/shopware'
+    if: github.repository == 'shopware/shopware' && github.ref_type == 'tag'
     env:
       GH_TOKEN: ${{ github.token }}
       CHECK_REPOSITORY: 'shopware/recipes'
       CHECK_WORKFLOW: 'nightly.yml'
       CHECK_STATUS: 'success'
+      DEBUG: ${{ runner.debug }}
```

Comment:
Most of our scripts set the `-x` option for the shell, when `DEBUG` is defined. So adding this allows us to get the full execution output when enabling debug logging via the GitHub UI.